data/test-cases/plugins/MS Teams sync/Slash commands/Link_channels.md

```diff
@@ -0,0 +1,46 @@
+---
+# (Required) Ensure all values are filled up
+name: "Link channel command"
+status: Active
+priority: Normal
+folder: Slash commands
+authors: "@AayushChaudhary0001"
+team_ownership: 
+- Change Team Name
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Run the `/msteams-sync link <msteams-teams-id> <msteams-channel-id>` command in the desired MM channel by the account of system admin.
```

Comment:
After the creation of the UI, the parameters for this command will change. So, we should add labels on the PR which adds UI and modifies this slash command.

Reply:
Why are we running the command here by system admin?